owasp-top10-2017-apps/a1/mongection/deployments/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+
+    server:
+        container_name: a1_mongection
+        build:
+            context: ../
+            dockerfile: deployments/Dockerfile
+        ports:
+            - "9995:9995"
+        networks:
+            - a1net
+        depends_on:
+            - mongo
+        external_links:
+            - mongo:mongo
+        restart: always
+
+    mongo:
```

Comment:
It appears that you're not using any kind of authentication for MongoDB, allowing any user to have access to all data stored in it. 🧐 What do you think about adding some credentials and using them when connecting to the database? You can check [this example](https://github.com/globocom/secDevLabs/tree/master/owasp-top10-2017-apps/a3/snake-pro/deployments) if you want to do so.

Reply:
Yeah I know this, but if you think that is better I can insert too without problems :D